step.sh

```diff
@@ -1,80 +1,31 @@
 #!/bin/bash
 set -e
 
-# Check that collector_config exists
-if [ ! -f "$collector_config" ]
-then
-  echo "Error: collector_config at $collector_config does not exist"
-  exit 1
-fi
-
-# Check that iglu eixsts
-if [ ! -f "$iglu" ]
-then
-  echo "Error: iglu at $iglu does not exist"
-  exit 1
-fi
-
-# Resolve the latest version if we need to
-echo "Request server version: $micro_version"
-if [[ "$micro_version" == "latest" ]]
-then
-  # See which tag the HTTP request is redirected to (can't use API due to unauthenticated rate limits)
-  micro_version=$(curl -I --silent https://github.com/snowplow-incubator/snowplow-micro/releases/latest | grep -iF location: | sed -E 's/\r$//' | sed -E 's/^.*tag\/(.*)$/\1/')
+httpschema="https"
+interface="snowplow-micro-ios.production.data.global-services.ckpd.co"
 
-  if [[ "$micro_version" == "" ]];
-  then
-    echo "Failed to detect latest version of Snowplow Micro"
-    exit 1
-  else
-    echo "Latest version is $micro_version"
-  fi
-fi
-
-# Create the directory for the server jar files if it doesn't already exist
-micro_dir="$HOME/.snowplow/micro"
-mkdir -p "$micro_dir"
+# Test the server
+echo "Resting the Snowplow Microcollector at $httpschema://$interface/micro/reset..."
+result=$(curl --silent "$httpschema://$interface/micro/reset")
 
-# Define where the .jar will live for the requested version
-jar="$micro_dir/snowplow-$micro_version.jar"
+# Parse the results
+total=$(jq -r '.total' <<< "$result")
 
-# Download the jar if it does not already exist
-if [ ! -f "$jar" ]
+# Check if there are any active events 
+if [[ $total -gt 0 ]]
```

Comment:
TIL `-gt 0` 👍 Maybe we can use these integer comparisons in the check-collector script too? I only compared strings originally because I didn't know bash 😬